docs/recent-releases/go-ipfs-0-6/README.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Overview
+---
+
+# Go-IPFS 0.6
+
+![The Go-IPFS logo.](./images/go-ipfs-logo.png)
+
+This is a relatively small release in terms of code changes, but it contains some significant changes to the IPFS protocol.
+
+## Features
+
+There are plenty of features packed into this release:
+
+- QUIC by default.
+- Libp2p Noise as a new security transport.
+- Custom 404 pages and base36 support.
+- Gossipsub upgrade.
+- New peering subsystem.
+
+## Breaking changes
+
+There are no breaking changes in this release. However, there is a migration will run, and the QUIC transport has changed protocol versions.
```

Comment:
this is grammatically incorrect ("there is a migration THAT will run", or "a migration will run"). Probably useful to indicate when and for what the migration will run